### ISSUE TITLE
WCAG2.1 validation fails with an error E626

### ISSUE THREAD

**Issue by caspioua** [highcharts/highcharts#13760]
Total Validator was used to check Highcharts accessibility compliance with the WCAG2.1 requirements
#### Expected behaviour
The chart widget complies the WCAG2.1 requirements

#### Actual behaviour
WCAG2.1 validation fails with the **E626**: All element names must be in lower case

![image](https://user-images.githubusercontent.com/46484910/85396508-8fa71f80-b55a-11ea-9376-f7ee2ce65263.png)

![image](https://user-images.githubusercontent.com/46484910/85396532-99308780-b55a-11ea-9343-97d1d32ca5c6.png)

#### Live demo with steps to reproduce
Just any line chart https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/line-basic/
![image](https://user-images.githubusercontent.com/46484910/85396881-3be90600-b55b-11ea-8ed4-fa97b4a9e063.png)

#### Product version
Highcharts v8.0.0

#### Affected browser(s)
any supported

**Comment by pawelfus** [comment]
Thanks for reporting the issue @caspioua 

I'm not sure what to do about this issue. This is SVG's namespace, where `clipPath` element should be `clipPath`, not `clippath` (W3: https://www.w3.org/TR/SVG11/masking.html#EstablishingANewClippingPath). WCAG 2.1 does not define this rule, it's in XHTML definitions: https://www.w3.org/TR/xhtml1/#h-4.2 

Perhaps it's a bug in total validator? E6262 is not an official WCAG 2.1 error and it seems xmlns is not correctly picked.

@caspioua - do you have similar errors using https://validator.w3.org/unicorn/ ? 

Any ideas @TorsteinHonsi @oysteinmoseng ?

**Comment by oysteinmoseng** [comment]
WCAG 2.1 requires in 4.1.1 - Parsing: `In content implemented using markup languages, elements have complete start and end tags, elements are nested according to their specifications, elements do not contain duplicate attributes, and any IDs are unique, except where the specifications allow these features.` There is no requirement for lowercase element names.

`clipPath` is the correct spelling of this element according to the SVG 1.1 specification, so this would be an issue with the Total Validator tool in validating SVG.
